packages/catalog-search/src/CurrentRefinements.jsx

```diff
@@ -14,13 +15,33 @@ import {
   NUM_CURRENT_REFINEMENTS_TO_DISPLAY,
   STYLE_VARIANTS,
   LEARNING_TYPE_PATHWAY,
+  LEARNING_TYPE_COURSE,
+  LEARNING_TYPE_PROGRAM,
 } from './data/constants';
 import {
   useActiveRefinementsAsFlatArray,
 } from './data/hooks';
 import { SearchContext } from './SearchContext';
 import { removeFromRefinementArray, deleteRefinementAction } from './data/actions';
 
+const messages = defineMessages({
+  [LEARNING_TYPE_COURSE]: {
+    id: 'search.facetFilters.filterTitle.course',
```

Comment:
We are not following Camel case format in IDs of strings. So I think, we can maintain the consistency as we are doing in learner portal and admin portal.
`search.facetFilters.filterTitle.course` -> `search.facet.filters.filter.title.course`

Reply:
I think it is better to use dot `.` to separate pages/components and use camelCase for page/component name.